packages/devextreme/testing/testcafe/helpers/accessibilityUtils.ts

```diff
@@ -1,14 +1,20 @@
 /* eslint-disable spellcheck/spell-checker */
 import { axeCheck, createReport } from '@testcafe-community/axe';
 
-export const a11yCheck = async (t: TestController, rules: object = {}): Promise<void> => {
-  const { error, results } = await axeCheck(t, undefined, {
-    rules: {
-      'color-contrast': { enabled: false },
-      'th-has-data-cells': { enabled: false },
-      ...rules,
-    },
-  });
+const defaultOptions = {
+  rules: {
+    'color-contrast': { enabled: false },
+    'th-has-data-cells': { enabled: false },
+  },
+};
+
+export const a11yCheck = async (
+  t: TestController,
+  selector: any = undefined,
```

Comment:
Should we create other overloaded method instead of editing the current one? I suggest set selector argument as the third in argument order.

Reply:
What other method overload are you talking about?

Reply:
I'm sorry, it is C# way) TS requires optional selector parameter instead as williamvinogradov wrote.